scripts/check-monitoring.go

```diff
@@ -0,0 +1,171 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+//go:generate go run check-monitoring.go ../monkit.lock
+
+// +build ignore
+
+package main
+
+import (
+	"fmt"
+	"go/ast"
+	"go/token"
+	"go/types"
+	"io/ioutil"
+	"os"
+	"sort"
+	"strings"
+
+	"golang.org/x/tools/go/packages"
+)
+
+var (
+	monkitPath    = "gopkg.in/spacemonkeygo/monkit.v2"
+	lockFilePerms = os.FileMode(0644)
+)
+
+func main() {
+	pkgs, _ := packages.Load(&packages.Config{
+		Mode: packages.NeedCompiledGoFiles | packages.NeedSyntax | packages.NeedName | packages.NeedTypes | packages.NeedTypesInfo,
+	}, "storj.io/storj/...")
+
+	var lockedFnNames []string
+	for _, pkg := range pkgs {
+		lockedFnNames = append(lockedFnNames, findLockedFnNames(pkg)...)
+	}
+	sortedNames := sortAndUnique(lockedFnNames)
+
+	outputStr := strings.Join(sortedNames, "\n")
+	if len(os.Args) == 2 {
+		ioutil.WriteFile(os.Args[1], []byte(outputStr+"\n"), lockFilePerms)
```

Comment:
what with returned error? I'm not sure what is our policy for scripts like that.

Reply:
Good point, we should handle the error; `// +build ignore` strikes again.